float32 brightnessValue
float32 gain
float32 exposure
bool autoParamAdj

int16 controlDigit
float32 autoSpeed
float32 manualSpeed
float32 omega
int8 controlMode
bool forward
bool backward